libcxx/include/__locale

```diff
@@ -1255,9 +1255,9 @@ extern template class _LIBCPP_EXTERN_TEMPLATE_TYPE_VIS codecvt_byname<char, char
 #if _LIBCPP_HAS_WIDE_CHARACTERS
 extern template class _LIBCPP_EXTERN_TEMPLATE_TYPE_VIS codecvt_byname<wchar_t, char, mbstate_t>;
 #endif
-extern template class _LIBCPP_DEPRECATED_IN_CXX20
+extern template class __attribute__((__libcpp_deprecated_in_cxx20()))
 _LIBCPP_EXTERN_TEMPLATE_TYPE_VIS codecvt_byname<char16_t, char, mbstate_t>; // deprecated in C++20
-extern template class _LIBCPP_DEPRECATED_IN_CXX20
+extern template class __attribute__((__libcpp_deprecated_in_cxx20()))
```

Comment:
Note that I had to use `__attribute__` here, since the `[[]]` version isn't allowed anywhere. This is the only place though.